src/poliastro/twobody/orbit.py

```diff
@@ -580,6 +580,32 @@ def geostationary(
         altitude = geo_radius - attractor.R
         return cls.circular(attractor, altitude)
 
+    @classmethod
+    @u.quantity_input(a=u.m, ecc=u.one, w=u.rad / u.s, period=u.s)
+    def heliocentric(cls, attractor, a, ecc, w=None, period=None):
+        if w is None and period is None:
+            raise ValueError("At least one among w or period must be passed")
+
+        if w is None:
+            w = 2 * np.pi / period
+
+        if a is None:
+            raise ValueError("The value of semi major axis is required")
+        mean_motion = np.sqrt(attractor.k / np.cube(a))
+        h = a * (1 - ecc ** 2)
+        theta = (-2 / 3) * ((h / attractor.k) ** 2) * (w / (mean_motion * attractor.J2))
+        inc = (180 / np.pi) * np.arccos(theta)
+
+        raan = (0 * u.deg,)
```

Comment:
I'm not sure why we're using tuples here, these probably make the code crash somewhere.

Reply:
That means that, apart from tests validating the numerical aspect, we would need something that at least uses the function.